src/components/ModalViews/DepositView/PrizeFrequencyBreakdown.tsx

```diff
@@ -0,0 +1,136 @@
+import { TransparentDiv } from '@components/TransparentDiv'
+import { usePrizePoolPrizes } from '@hooks/v4/PrizePool/usePrizePoolPrizes'
+import { usePrizePoolTicketTwabTotalSupply } from '@hooks/v4/PrizePool/usePrizePoolTicketTwabTotalSupply'
+import { useSelectedPrizePool } from '@hooks/v4/PrizePool/useSelectedPrizePool'
+import { Amount } from '@pooltogether/hooks'
+import {
+  formatCurrencyNumberForDisplay,
+  formatDailyCountToFrequency
+} from '@pooltogether/utilities'
+import { TimeUnit } from '@pooltogether/utilities/dist/types'
+import { PrizeTierV2 } from '@pooltogether/v4-utils-js'
+import classNames from 'classnames'
+import { useMemo } from 'react'
+import { useTranslation } from 'react-i18next'
+import { LoadingPrizeRow, PrizeTableCell, PrizeTableHeader } from './ExpectedPrizeBreakdown'
+
+export const PrizeFrequencyBreakdown = (props: { prizeTier: PrizeTierV2; className?: string }) => {
+  const { prizeTier, className } = props
+  const prizePool = useSelectedPrizePool()
+  const { data: prizeData, isFetched: isPrizesFetched } = usePrizePoolPrizes(prizePool)
+  const { data: ticketSupply, isFetched: isTicketSupplyFetched } =
+    usePrizePoolTicketTwabTotalSupply(prizePool)
+  const { t } = useTranslation()
+
+  const isFetched = isPrizesFetched && isTicketSupplyFetched
+
+  const calculateDprMultiplier = (
+    rawDpr: number,
+    ticketSupplyAmount: string,
+    totalPrizeAmount: string
+  ) => {
+    const dpr = rawDpr / 10 ** 9
+    const tvl = parseFloat(ticketSupplyAmount)
+    const prizes = parseFloat(totalPrizeAmount)
+    return (dpr * tvl) / prizes
+  }
+
+  const prizeChances = useMemo(() => {
```

Comment:
This data should be the same as what is returned from `usePrizePoolExpectedPrizes`, we should reuse that (or fix it if it's wrong)

Reply:
You're correct - and I wish I had realized that sooner... Oof.

Just checked the data and it all seems to match. Updated in e002259